dataSourceImpl/build.gradle

```diff
@@ -14,6 +14,7 @@ android {
     versionName "1.0"
 
     buildConfigField "String", "APP_PREFERENCES", '"settings"'
+    buildConfigField "String", "TOKEN", '"token"'
```

Comment:
Too complicated, It would be better to declare this variable in `companion object` of `DatastorePreferencesImpl`

Reply:
+